kalite/control_panel/tests/control_panel.py

```diff
@@ -107,6 +107,34 @@ def test_teachers_have_no_facility_delete_button(self):
         with self.assertRaises(NoSuchElementException):
             self.browser.find_element_by_xpath('//a[@class="facility-delete-link"]')
 
+    def test_facility_with_no_missing_metadata(self):
+        facility_name = 'no-missing-metadata'
+        self.fac = self.create_facility(name=facility_name)
```

Comment:
I realized creating a new facility (via add new facility on management form) initially assigns certain fields to an empty string. Not sure if that's the intended behavior, but for now I've updated the check_metadata function to check for both null or empty strings.